ios/FluentUI.Demo/FluentUI.Demo/Demos/ButtonDemoController.swift

```diff
@@ -23,13 +23,42 @@ class ButtonDemoController: DemoController {
             disabledButton.setTitle("Button", for: .normal)
 
             addRow(items: [button, disabledButton], itemSpacing: 20)
+
+            if let image = style.image {
+                let iconButton = Button(style: style)
+                iconButton.setTitle("Button", for: .normal)
+                iconButton.image = image
+
+                let disabledIconButton = Button(style: style)
+                disabledIconButton.isEnabled = false
+                disabledIconButton.setTitle("Button", for: .normal)
+                disabledIconButton.image = image
+
+                addRow(items: [iconButton, disabledIconButton], itemSpacing: 20)
+
+                let iconOnlyButton = Button(style: style)
```

Comment:
I don't see iconOnlyButton and disabledIconOnlyButton on the screenshots. Were they added after you took the screenshots?

Reply:
I updated the screenshots.